view/template/trial.hbs

```diff
@@ -69,6 +69,23 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                <div class = "modal-body">
+                    <form id = "check-compliance" method= "GET">
+                        <div class = "form-group">
+                            <div class = "col-md-4">
+                                <label>From date:-</label>
+                            </div>
+                            <input id="fromDate" name="fromDate" type="date" value="{{ fromDate }}" >
+                        </div>
+                        <div class = "form-group">
+                            <div class = "col-md-4">
+                                <label>To date:-</label>
+                            </div>
+                            <input id="toDate" name="toDate" type="date" value="{{ toDate }}">
+                        </div>
+                        <input type = "Submit" value = "Submit"></input>
```

Comment:
did you call the `submit` function from the `focus` event, the `change` event, or both?